src/board-explorer-app/board-explorer-app.html

```diff
@@ -1790,7 +1790,9 @@
       console.log("Asking for components...");
       var components = [];
       this.$.boardViewer.getComponents().forEach(function(refdes) {
-        components.push(this.$.boardViewer.getInfo(refdes));
+        if (refdes != 'JTAGSEL') {
+          components.push(this.$.boardViewer.getInfo(refdes));
+        }
       }.bind(this));
```

Comment:
@juliocri The board-explorer shouldn't have board specific checks in it. Was this left over from when you were investigating how to fix the issue?